gradle.properties

```diff
@@ -3,11 +3,11 @@
 org.gradle.jvmargs=-Xms32M -Xmx4G -XX:+UseG1GC -XX:+UseStringDeduplication
 
 loader_version=0.14.21
-viaver_version=4.7.0-1.20-rc1-SNAPSHOT
+viaver_version=4.7.0
 yaml_version=2.0
 
-publish_mc_versions=1.19.4, 1.18.2, 1.17.1, 1.16.5, 1.15.2, 1.14.4, 1.8.9
-# example: 1.19.1-rc2
-modrinth_mc_snapshot=1.20-rc1
-# example: 1.19-Snapshot
-curseforge_mc_snapshot=1.20-Snapshot
+publish_mc_versions=1.20, 1.19.4, 1.18.2, 1.17.1, 1.16.5, 1.15.2, 1.14.4, 1.8.9
+# example: 1.19.1
+modrinth_mc_snapshot=1.20
```

Comment:
Just make it blank

Reply:
on it.

Reply:
Removed it but github isn't displaying the change for some reason.